ddtrace/appsec/_asm_request_context.py

```diff
@@ -121,6 +129,18 @@ def flush_waf_triggers(env: ASM_Environment) -> None:
         else:
             root_span.set_tag(APPSEC.JSON, json.dumps({"triggers": report_list}, separators=(",", ":")))
         env.waf_triggers = []
+    telemetry_results = get_value(_TELEMETRY, _TELEMETRY_WAF_RESULTS)
+    if telemetry_results:
+        from ddtrace.appsec._metrics import DDWAF_VERSION
+
+        root_span.set_tag_str(APPSEC.WAF_VERSION, DDWAF_VERSION)
+        if telemetry_results["total_duration"]:
+            update_span_metrics(root_span, APPSEC.WAF_DURATION, telemetry_results["duration"])
```

Comment:
is it possible for the waf to run multiple times, so we do increase the duration and total duration on a span more than once?

I am wondering if you also need a count for how many times we updated.

e.g. total duration of 100ms is bad if the waf ran once, but might be fine if it ran 10 times?

Reply:
we seem to do that for the RASP, right?

Reply:
yeah, but do we want something like `sum_eval` for the WAF like we do for the RASP?